scripts/update-crd-reference/main.sh

```diff
@@ -2,16 +2,22 @@
 
 set -e
 
-CRD_DOCS_GENERATOR_VERSION=0.4.0
+CRD_DOCS_GENERATOR_VERSION=0.4.0-ee74b086d228a80d7c560ffe5a90bf7480c98cd7
```

Comment:
This depends on https://github.com/giantswarm/crd-docs-generator/pull/33

TODO: After crd-docs-generator is released, update the version here.